src/unity/CMakeLists.txt

```diff
@@ -71,20 +71,20 @@ endif()
 ############################################################
 
 if (WIN32 OR is_xcode)
-  group_sources(src)
-  group_sources(extras/rippled/src)
-  group_sources(extras/rippled/Builds)
+  group_sources_in("${CMAKE_CURRENT_SOURCE_DIR}/../../" src)
+  group_sources_in("${CMAKE_CURRENT_SOURCE_DIR}/../../" extras/rippled/src)
+  group_sources_in("${CMAKE_CURRENT_SOURCE_DIR}/../../" extras/rippled/Builds)
 endif()
 
 ############################################################
 
 add_with_props(lib_src ripple-libpp.cpp
-  -I"${CMAKE_SOURCE_DIR}/"extras/rippled/src/secp256k1
+  -I"${CMAKE_CURRENT_SOURCE_DIR}/"../../extras/rippled/src/secp256k1
```

Comment:
the quotes here are a little odd...it produces this in the build:

```
-I"/Users/ellery/work/ripple-libpp/"extras/rippled/src/secp256k1 -I"/Users/ellery/work/ripple-libpp/"extras/rippled/src/ed25519-donna
```
..which I guess works fine but looks strange. Should we just wrap the entire string in quotes or possibly omit them completely?